spec/fabricators/feedback_fabricator.rb

```diff
@@ -0,0 +1,2 @@
+Fabricator(:feedback) do
+end
```

Comment:
Feedback objects are expected to have an email and comment. They should probably be added here. That being said, you don't appear to be using this fabricator so you could also just remove it.